bonsai/ethereum-relay/Cargo.toml

```diff
@@ -51,6 +52,7 @@ validator = { version = "0.16", features = ["derive"] }
 bincode = "1"
 bytemuck = "1.13"
 risc0-zkvm-methods = { path = "../../risc0/zkvm/methods", default-features = false }
+rstest = "0.18"
```

Comment:
https://docs.rs/rstest/latest/rstest/ interesting! TIL, @flaub this looks like a interesting library for testing.

Reply:
You can see it's usage [here] for example.

[here]: https://github.com/risc0/risc0/pull/903/files#diff-e45dd1869f09cca9bbdd8d39cd54d53427049fc3c256e3265f7497921ca5397bR88-R107